src/main/java/com/google/codeu/data/Clinic.java

```diff
@@ -0,0 +1,31 @@
+package com.google.codeu.data;
+import com.opencsv.bean.CsvBindByName;
+import java.util.Arrays;
+
+public class Clinic {
```

Comment:
I think it'll be better to add some comments

Reply:
Good suggestion!